docs/multicluster/user-guide.md

```diff
@@ -3,20 +3,29 @@
 ## Table of Contents
 
 <!-- toc -->
-- [Quick Start](#quick-start)
-- [Installation](#installation)
-  - [Preparation](#preparation)
-  - [Deploy Antrea Multi-cluster Controller](#deploy-antrea-multi-cluster-controller)
-  - [Create ClusterSet](#create-clusterset)
-- [Multi-cluster Gateway Configuration](#multi-cluster-gateway-configuration)
-- [Multi-cluster Service](#multi-cluster-service)
-- [Multi-cluster Pod to Pod Connectivity](#multi-cluster-pod-to-pod-connectivity)
-- [Multi-cluster NetworkPolicy](#multi-cluster-networkpolicy)
-  - [Egress Rule to Multi-cluster Services](#egress-rule-to-multi-cluster-service)
-  - [Ingress Rule](#ingress-rule)
-- [ClusterNetworkPolicy Replication](#clusternetworkpolicy-replication)
-- [Build Antrea Multi-cluster Controller Image](#build-antrea-multi-cluster-controller-image)
-- [Known Issue](#known-issue)
+- [Antrea Multi-cluster User Guide](#antrea-multi-cluster-user-guide)
```

Comment:
I don't see changes in section titles, why is there so much diff in the toc?

Reply:
The contents were generated automatically by my IDE plugin, revised.